specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Note that this is a data model, which is only meant to characterize relationships between types, not how they are organized or stored.  Perhaps this conceptual perspective is the source of the confusion I have about some of your earlier comments.   The only point of this concept is to (conceptually, not necessarily actually) aggregate all CVRs uploaded by counties.